src/firebase-db.ts

```diff
@@ -25,6 +25,14 @@ export type FirebaseAppName = "report-service-dev" | "report-service-pro";
 
 let portalData: IPortalData | IAnonymousPortalData | null;
 
+// const validFsId = (anyPath: string) => {
```

Comment:
Did you mean to leave in this commented-out code?

Reply:
No, forgot about that. Thanks. It's gone now.